addons/editor-theme3/addon.json

```diff
@@ -14,6 +14,10 @@
   "dynamicDisable": true,
   "injectAsStyleElt": true,
   "versionAdded": "1.2.0",
+  "latestUpdate": {
+    "version": "1.23.0",
+    "newSettigs": ["comment-color"]
```

Comment:
Either I did something wrong here or the new tag just doesn't display next to colour inputs for some reason.

Reply:
typo